### DISCUSSION TITLE
Cellrank on partially differentiated cells dataset

### DISCUSSION THREAD

**Discussion by castaway1990** [theislab/cellrank#509] [Method/Algorithm] [answered]
Hi everyone,
I am performing SC analyses on brain developmental models. In my experimental design we have cells at several differentiation stages and I am trying to use Cellrank to define initial and terminal states in timepoint-wise manner coupled with the relative absorption paths. 
After some trials I asked myself what is fair to expect from terminal states identification in datasets from extremely plastic state cells (e.g. mix of non differentiated, partially differentiated and completely differentiated cells).  
I am wondering if  i) Cellrank would mark as dataset-specific terminal state also partially differentiated population detaching towards something not yet present at the current timepoint or ii) it would prioritise more differentiated states and consider the partially differentiated ones as intermediate states.

Thanks a lot,
Davide

**Comment by Marius1311** [comment, marked as answer]
Hi @castaway1990, sorry for only getting back to you now - I just realized that we don't get notifications for new posts in the Discussions section - so it's probably saver to post questions as issues because we get notified then. 

Anyways, regarding your questions: 
1. If you restrict CellRank to a certain time-point, it will attempt to find terminal states restricted to that time-point - if it turns out that the most mature state in that time-point is really only partially differentiated with respect to the entire biological trajectory, maybe you could attempt applying CellRank to the integrated dataset, across time-points?
2. In an integrated dataset, I would expect that CellRank correctly tells apart intermediate from terminal states.

**Reply by castaway1990** [reply]
No problem about delay @Marius1311, thanks for the answer, very useful indeed. I will try on the integrated dataset for sure. So far I'm keeping a part the sequential passages to highlight the step dynamics. I'll do some more tests and in case i'll get back to you, for now thanks!.

Davide

**Reply by Marius1311** [reply]
You're welcome!
